### PR TITLE
[PATRON2020-167] - BlindsController - rounding bug fixed, stepper controlled by gray code logic

### PR DESCRIPTION
**Pull request by kuba8686**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,4 @@
-output/
-.vscode/
-*.py
+output/
```
Comment (Siwy160) [inline .gitignore]: What happened here?

Comment (kuba8686) [reply .gitignore]: I didnt make any changes in .gitignore.
dont know why is like that.
Anyway fixed - .gitignore is not anymore as a changed file

**Comment by kuba8686** [comment]
Added gray code logic for control step mottor.

**Comment by cichy1024** [comment]
I'll wait for other approvals.

File: BlindsController/blinds.h [hunk 2]
```diff
@@ -14,39 +14,30 @@ static int currentVal = 0;     /* innitial val for position of blinds (stepper)
...
-        {0, 0, 0, 1},
-        {0, 0, 1, 1},
-        {0, 0, 1, 0},
-        {0, 1, 1, 0},
-        {0, 1, 0, 0},
-        {1, 1, 0, 0},
-        {1, 0, 0, 0},
-        {1, 0, 0, 1}
-    };
+    const int step_sequence[] = {0, 1, 3, 2};
```
[the remark this answers is not in the document]
Comment (kuba8686) [reply BlindsController/blinds.h]: Done

File: BlindsController/blinds.h [hunk 3]
```diff
@@ -14,39 +14,30 @@ static int currentVal = 0;     /* innitial val for position of blinds (stepper)
...
     
-    digitalWrite(stepper_pin[0], stepPinVal[_step][0]);  
-    digitalWrite(stepper_pin[1], stepPinVal[_step][1]); 
-    digitalWrite(stepper_pin[2], stepPinVal[_step][2]); 
-    digitalWrite(stepper_pin[3], stepPinVal[_step][3]);
-        
+    digitalWrite(stepper_pin[0], step_sequence[_step] >> 1);
+    digitalWrite(stepper_pin[1], step_sequence[_step] & 0x1);
+    digitalWrite(stepper_pin[2], (3 - step_sequence[_step]) >> 1);
+    digitalWrite(stepper_pin[3], (3 - step_sequence[_step]) & 0x1);
```
[the remark this answers is not in the document]
Comment (kuba8686) [reply BlindsController/blinds.h]: Done

**Pull request merged**